pkg/deploy/deploy.go

```diff
@@ -179,6 +179,11 @@ func (d *deployer) configureDNS(ctx context.Context) error {
 		return err
 	}
 
+	portalPip, err := d.publicipaddresses.Get(ctx, d.config.ResourceGroupName, "portal-pip", "")
```

Comment:
Did you tested any of this deploy code or this is "fix in prod" case?

Reply:
No - want to try INT this week.